keycloak/keycloak-admin-ui
# Advanced tab

## Motivation

Introduced advanced tab for the client section [marvel](https://marvelapp.com/prototype/9a93gef)

## Verification Steps

Open the details of a client and see that it now has an extra tab called "Advanced"
The clients of type "openid-connect" have a different tab then the once that are of type SAML

## Checklist:


## Additional Notes
